Dockerfile.dapper

```diff
@@ -1,11 +1,11 @@
-FROM golang:1.16.13-alpine3.15
+FROM registry.suse.com/bci/golang:1.16
```

Comment:
Should we move to the 1.17 image of Go, since 1.16 is EOL? The module itself, by the way, is declared to be built with Go 1.15, but that's fine?

Reply:
@maxsokolovsky Updating's Go version from 1.15 to 1.17 is not directly in the scope of the PRs related to migration to SLE-BCI.

In practice we can do it, but I'm hesitant as this can add further variables in the process.